app/src/main/java/com/example/concordia_campus_guide/Fragments/LocationFragment/LocationFragmentViewModel.java

```diff
@@ -147,7 +150,7 @@ public void addBuildingMarker(GoogleMap map, LatLng centerPos, String buildingLa
                         .anchor(0.5f,0.5f)
                         .alpha(0.90f)
                         //This line should be included whenever we test the UI for the marker:
-                        .title(buildingLabel)
+                        //.title(buildingLabel)
```

Comment:
Is this the proper way of doing the Ui test? We might forget to uncomment it.

Reply:
idk, this is not me, this is part of the building marker story

Reply:
I am aware of this issue. unfortunately, I didn't have the time to investigate about it yet .